invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchApp.js

```diff
@@ -166,6 +176,7 @@ SearchApp.defaultProps = {
       listView: true,
       gridView: false,
     },
+    defaultSortingOnEmptyQueryString: {},
```

Comment:
Minor: Should we actually provide a default value here since there is no default value in ```react-seachkit``` either. Or even better in the ```SearchAppInvenioRestConfigHelper``` :)

Reply:
It is added to the default generation <https://github.com/inveniosoftware/invenio-search-ui/pull/111/files#diff-d093386b38c5478a02f6a187974a9d87c86e5b239dada52a762b50c89758dca1R257>

Personally I have nothing against removing it. It is just there for completeness. @ntarocco might have more insight related to rsk.

Reply:
You can indeed remove it if you prefer, not needed. We were just following the same pattern as `initialQueryState: {},`.